package.nls.cs.json

```
@@ -15,28 +15,28 @@
-    "mql_tools.commands.InsImport.title": "Vložit DLL/EX5 jako #import",
-    "mql_tools.commands.InsTime.title": "Vložit datum a čas jako RRRR.MM.DD hh:mm:ss",
-    "mql_tools.commands.InsIcon.title": "Vložit ikonu jako #property",
-    "mql_tools.commands.openInME.title": "Otevřít v 'MetaEditor'",
-    "mql_tools.commands.openTradingTerminal.title": "Otevřít obchodní terminál",
-    "mql_tools.commands.createCommentary.title": "Vytvořit komentář funkce",
-    "mql_tools.commands.reportNoisyDiagnostic.title": "Nahlásit hlučnou diagnostiku clangd",
-    "mql_tools.commands.runBacktest.title": "Spustit backtest",
-    "mql_tools.commands.startDebugging.title": "Spustit ladicí relaci MQL",
-    "mql_tools.commands.stopDebugging.title": "Zastavit ladicí relaci MQL",
-    "mql_tools.commands.showStartupPage.title": "Zobrazit uvítací stránku"
+    "mql_tools.configuration.AutoVersionBump.description": "Automatically bump the minor version in #property version before each compile.","mql_tools.configuration.VersionConstantNames.description": "Names of const string variables to auto-bump on compile (e.g. EA_VERSION).","mql_tools.commands.compileFile.title": "Kompilovat soubor MQL",
```

Biome (2.4.15)

[error] The key mql_tools.configuration.AutoVersionBump.description was already declared.

(lint/suspicious/noDuplicateObjectKeys)